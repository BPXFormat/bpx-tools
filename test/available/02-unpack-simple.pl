$Test = {
    Name => "Unpack (SIMPLE)",
    Command => "-f test/available/test.bpx unpack",
    Description => "Test the unpack command",
    Status => 0
};

sub TestBegin {

}

sub TestEnd {
    my $res = EnsureEqual("LICENSE_LF.txt", "test/LICENSE_LF.txt");
    unlink("LICENSE_LF.txt");
    unlink("test/LICENSE_LF.txt");
    return $res;
}
