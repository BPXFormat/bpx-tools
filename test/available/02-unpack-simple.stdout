Reading LICENSE_LF.txt with 1517 byte(s)...
